components/authn-service/tokens/pg/adapter.go

```diff
@@ -83,23 +84,25 @@ func (a *adapter) UpdateToken(ctx context.Context,
 	if projects == nil {
 		projects = []string{}
 	}
+	var row *sql.Row
 	if description != "" {
-		if err := a.db.QueryRowContext(ctx,
-			`UPDATE chef_authn_tokens cat SET active=$2, description=$3, project_ids=$4, updated=NOW() 
+		row = a.db.QueryRowContext(ctx,
+			`UPDATE chef_authn_tokens cat
+			SET active=$2, description=$3, project_ids=$4, updated=NOW() 
```

Comment:
as a side note, we should move this `now()` out of the code . . . right? I'm curious why we don't let the DB handle it